rules/Multifactor-Google-Authenticator-Do-Not-Rename.js

```diff
@@ -16,19 +17,22 @@ function (user, context, callback) {
         return callback(null, user, context);
     }
 
-    getUserProfile(user)
-        .then(function (user_profile) {
-            if (!loggedInRecently(user_profile)) {
-                enableMFA(context);
-            }
-
-            return callback(null, user, context);
-        })
-        .catch(function(err) {
-            enableMFA(context);
-            return callback(null, user, context);
-        });
-
+    // Skip MFA if user's IP is on a corporate network
+    var CORPORATE_NETWORK_IPS = [
+        // https://github.com/ministryofjustice/moj-ip-addresses/blob/master/moj-cidr-addresses.yml
+    ];
+    var userIsOnCorporateNetwork = CORPORATE_NETWORK_IPS.some(
+    function (ip) {
```

Comment:
[Apparently](https://github.com/ministryofjustice/analytics-platform-auth0/pull/31/files#r202975282) ES6 can be used in Auth0 rules.

If that's the case you may prefer to use [`Array#includes()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) so this could become:

```javascript
CORPORATE_NETWORK_IPS.includes(context.request.ip)
```

👆 not tested ⚠️

Reply:
[Auth0 provides Node 8 as of April 2018](https://auth0.com/docs/migrations/guides/extensibility-node8#summary-of-the-migration)